### ISSUE TITLE
Sync version of readJSON()?

### ISSUE THREAD

**Issue by rauschma** [ljharb/node-json-file#11]
For many command line scripts, a sync version is enough. Add one? E.g.:

```js
readJSON.JSONFile = JSONFile;
readJSON.sync = readJSONSync;
module.exports = readJSON;
```

**Comment by ljharb** [comment]
hmm - I intentionally didn't add it originally, because I generally think that nobody should ever use sync APIs when there's a sync option available.

That said, it wouldn't be that hard to add one - I'll consider it :-)

**Comment by rauschma** [comment]
Use case: shell scripting (including static generation of web sites). Then I prefer the sync versions, because the code is simpler. You don’t gain anything by being async.

**Comment by ljharb** [comment]
Thanks for explaining in more detail. I'm not sure that's true that you don't gain anything - I think expressing all operations, sync or async, as a promise/observable dependency tree both allows the engine the ability to optimize things better, as well as makes code cleaner, even for shell scripting.

I'll likely end up adding this though, after a refactor I intend to do today.

**Comment by ljharb** [comment]
Released as `v3.2.0`.
